test/components/componentTests.ts

```diff
@@ -78,6 +78,17 @@ describe("Component", () => {
       svg.remove();
     });
 
+    it("remove dom elements in previous svg element when switching element", () => {
```

Comment:
change sentence to "removes DOM elements in previous svg when anchoring to a different svg"